cli/src/lib.rs

```diff
@@ -655,6 +655,22 @@ fn restore_toolchain(restore_cbs: RestoreToolchainCallbacks) -> Result<()> {
     Ok(())
 }
 
+/// Get the system's default license - what 'npm init' would use.
+fn get_npm_init_license() -> Result<String> {
+    let npm_init_license_output = std::process::Command::new("npm")
+        .arg("config")
+        .arg("get")
+        .arg("init-license")
+        .output()?;
```

Comment:
I doubt anyone actually changes this. Mine was set to `ISC`, which I believe is the default. Maybe it would be better to just use Anchor's [license](https://github.com/coral-xyz/anchor/blob/81c8c556e8dffc6c0fcc23b4025d35508f19a59a/LICENSE) (Apache-2.0) as the default? Many Solana ecosystem projects already use that, including the official Solana projects.

Reply:
True, a few people set it to UNLICENSED or MIT, but it's not super common to change. The main thing is it's the 'official' way to set the default license `npm` uses, so it seems good to be consistent with the most popular way to make an npm module, rather than just decide a preferred license. If someone *did* set their own license, especially if it was something that wasn't OSS, then Apache-2.0 would be quite a surprise.

Reply:
Thanks, I don't mind either way.